finality-aleph/src/network/manager/service.rs

```diff
@@ -596,15 +597,19 @@ impl<NI: NetworkIdentity, D: Data> Service<NI, D> {
     }
 }
 
-/// Input/output interface for the connection manager service.
-pub struct IO<D: Data, M: Multiaddress, VN: ValidatorNetwork<M::PeerId, M, DataInSession<D>>>
-where
-    M::PeerId: PublicKey,
+/// Input/output interface for the connectiona manager service.
```

Comment:
```suggestion
/// Input/output interface for the connection manager service.
```

Reply:
Demariofied.

Reply:
I'm pretty sure I already fixed this in my PR before hmmmm